content/get-haiku/r1beta4/release-notes.md

```diff
@@ -0,0 +1,189 @@
+
+Please keep in mind that this is beta-quality software, which means it is feature complete but still contains known and unknown bugs. While we are increasingly confident in its stability, we cannot provide assurances against data loss.
+
+For most of this release cycle, waddlesplash was [employed as a contractor](https://www.haiku-os.org/news/2021-08-25_hiring_waddlesplash/) to work on Haiku. His contract is presently ongoing, supported by the generous donations of readers like you to Haiku, Inc., a 501(c)3 non-profit.
+
+*To download Haiku or learn how to upgrade from R1/beta3, see "[Get Haiku!](/get-haiku/)". For press inquiries, see "[Press contact](#press-contact)".*
+
+## System requirements
+
+This release is available on x86 32-bit and 64-bit platforms.
+
+{{< alert-info "BeOS R5 Compatibility" "Note that BeOS R5 compatibility is only provided on the 32-bit images." >}}
+
+<table><tr><td>
+<h4>MINIMUM (32-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Pentium II; AMD Athlon</li>
+<li><strong>Memory:</strong> 384MB</li>
+<li><strong>Monitor:</strong> 800x600</li>
+<li><strong>Storage:</strong> 3GB</li>
+</ul>
+</td><td>
+<h4>RECOMMENDED (64-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Core i3; AMD Phenom II</li>
+<li><strong>Memory:</strong> 2GB</li>
+<li><strong>Monitor:</strong> 1366x768</li>
+<li><strong>Storage:</strong> 16GB</li>
+</ul>
+</td></tr></table>
+
+<!--{{< alert-info "SSE2 support" "SSE2 support is required to use the WebPositive web browser. On machines where this is not available, it is recommended to install the NetSurf browser instead." >}}-->
+
+## New features
+
+### Improved HiDPI support
+
+<a href="/files/blog/waddlesplash/2022_09-unscaled.png"><img src="/files/blog/waddlesplash/2022_09-unscaled.png" width="49%"/></a>
+<a href="/files/blog/waddlesplash/2022_09-hidpi-after.png"><img src="/files/blog/waddlesplash/2022_09-hidpi-after.png" width="49%"/></a>
+<i><small>On the left: various applications on Haiku at standard DPI. On the right: those same applications, at "200%" scale (24pt font size.)</small></i>
+
+Previous releases of Haiku partially supported HiDPI scaling merely by changing the system font size; however, not all metrics were actually tied to the font size as they should be, and not all applications actually obeyed the system-set font size. Much work over the past release cycle has gone into rectifying these problems, and as you can now see in the above screenshots, most native applications now scale sufficently well. There are still some weak spots, but these should be resolved in time.
+
+Additionally, on first boot Haiku now attempts to automatically detect if you are on a HiDPI display and set appropriate sizes. (These can of course be adjusted in Appearance preferences.) For now, it is not possible for changes to these settings to take effect without a reboot.
```

Comment:
I would remove the parenthesis in the second sentence, to make it clear how and where people can manually tweak their HiDPI settings.

Reply:
Maybe make it even more concrete with:
> These can of course be adjusted with the font size settings of the Appearance preferences.

Otherwise people may look for some HiDPI or "Zoom" setting.